modules/gatk4/createsomaticpanelofnormals/main.nf

```diff
@@ -24,18 +24,18 @@ process GATK4_CREATESOMATICPANELOFNORMALS {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
+
     def avail_mem = 3
     if (!task.memory) {
         log.info '[GATK CreateSomaticPanelOfNormals] Available memory not known - defaulting to 3GB. Specify process memory requirements to change this.'
     } else {
         avail_mem = task.memory.giga
     }
     """
-    gatk --java-options "-Xmx${avail_mem}g" \\
-        CreateSomaticPanelOfNormals \\
-        -R $fasta \\
-        -V gendb://$genomicsdb \\
-        -O ${prefix}.vcf.gz \\
+    gatk --java-options "-Xmx${avail_mem}g" CreateSomaticPanelOfNormals \\
+        --variant gendb://$genomicsdb \\
```

Comment:
I guess a GATK thing to only use this DB?

Reply:
I'd assume so